Wrappers/Python/test/test_quality_measures.py

```diff
@@ -50,9 +51,22 @@ def setUp(self):
 
             self.dc1 = dc1
             self.dc2 = dc2
+            
+            self.mask=ig.allocate(0)
+            self.mask.array[:50,:50]=1
```

Comment:
The use of `.array` to get to the numpy array in a datacontainer is ok here where we know that this is a CIL DataContainer not a SIRF one